src/import/import-fs.c

```diff
@@ -286,6 +330,31 @@ static int parse_argv(int argc, char *argv[]) {
                         arg_read_only = true;
                         break;
 
+                case ARG_DIRECT:
+                        arg_direct = true;
```

Comment:
why is this a flag while the other are booleans?

Reply:
i guess i could also make it a boolean. But my thinking was, that the booleans tweak details about the output, but they don't alter the operation as a whole. the direct mode is different: it changes the operation executed to something quite different. it's like an adjective to the verb that turns the verb into something related but sufficiently different.

not sure if that is enough to make it not a bool...

Reply:
that sounds fine, was just wondering